tez-tests/pom.xml

```diff
@@ -124,11 +124,34 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>org.bouncycastle</groupId>
       <artifactId>bcprov-jdk15on</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>2.18.0</version>
```

Comment:
this test-scoped dependency is included twice in this pom, is that intentional? also, its version differs from the one in the root pom (4.3.1), can you place a comment here about that?

Reply:
You are right. I missed this declaration. Will make this change.